src/main/java/jpo/sdw/depositor/depositors/SDWDepositor.java

```diff
@@ -25,10 +25,11 @@ public void deposit(String message) {
          HttpHeaders headers = new HttpHeaders();
          headers.setContentType(MediaType.APPLICATION_JSON);
 
-         HttpEntity<String> httpEntity = new HttpEntity<String>(message ,headers);
-         
-         ResponseEntity<String> result = this.getRestTemplate().postForEntity(this.getDestination(), httpEntity, String.class);
-         
+         HttpEntity<String> httpEntity = new HttpEntity<String>(message, headers);
+
+         ResponseEntity<String> result = this.getRestTemplate().postForEntity(this.getDestination(), httpEntity,
+               String.class);
+
          logger.info("Response received. Status: {}, Body: {}", result.getStatusCode(), result.getBody());
```

Comment:
Since we're using API Keys now, I'm concerned the user who configures this could revoke his API Key and the ODE would fail to deposit into the SDX.

If we get back a 401 here, the user likely revoked his API Key or it expired (though I doubt we'd configure this with an expiring API Key).

If we get back a 403, the user doesn't have authority to deposit the TIM into the specified service region.

This may be outside the scope of this PR, but if the we get a 401 or 403 here, should we send out a notification of some sort, rather than just logging it? 

Another thing to consider: which user will be provisioning an API key for use here? Will we have a "ODE User" account that we don't use, except to provision an API key for the ODE to use? And maybe this ODE User has the ability to deposit a TIM anywhere?

Reply:
I've added additional logging and email for error to the October 2 commit. Note that we will provision a dedicated user and API Key for the ODE to use that will be able to deposit a TIM anywhere and have no expiration date.